flask_app.py

```diff
@@ -1,3 +1,32 @@
-"""
-Put your Flask app code here.
-"""
\ No newline at end of file
+from flask import Flask, render_template, request
+import os
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return render_template('index.html')
+
+
+@app.route('/result/', methods=['POST', 'GET'])
+def result():
+    error = None
+    if request.method == 'POST':
+        result = request.form
+        for key, val in result.items():
```

Comment:
This loop works but there should be a cleaner way to do this; I would probably just explicitly access these with dictionary keys. The server won't be happy if you send a POST request that doesn't have one of those fields.